docs/gulpfile.js

```diff
@@ -140,4 +151,20 @@ module.exports = function(gulp, IS_RELEASE_BUILD) {
       .pipe(gulp.dest('dist/docs/js'));
   });
 
+  gulp.task('docs-karma', ['docs-js'], function(done) {
```

Comment:
Let's not use the global karma config for docs testing. 
I think docs testing should be explicitly run separate of library/source/component testing.